### PR TITLE
Sync regex

### PR DESCRIPTION
**Pull request by gintunas**
Sync regex idea with current master

### CODE REVIEW

File: Mood_Busters/RegexStringCheck.cs [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
-    class RegexStringCheck : IRegex
+    static class RegexStringCheck
     {
         private const string pattern = "((Error|Unknown|Happy|Sad|Angry|Confused|Disgusted|Surprised|Calm|Fear) (?:100|[1-9]?[0-9])%)";
-        Regex rx = new Regex(pattern);
-        public bool checkString(string text)
+        public static bool checkString(string text)
         {  
-            return rx.IsMatch(text);
+            return (new Regex(pattern)).IsMatch(text);
```
Comment (pauliux00) [inline Mood_Busters/RegexStringCheck.cs]: Why is a new regex object being created every time the check function is called?

Comment (gintunas) [reply Mood_Busters/RegexStringCheck.cs]: ok

Comment (gintunas) [reply Mood_Busters/RegexStringCheck.cs]: Fixed

**Comment by pauliux00** [comment]
Looks good!

**Pull request merged**